### PR TITLE
Multiple updates to docker-centos

### PR DESCRIPTION
**Pull request by ashcrow**
- split commands for easier reading
- verify install of packages
- labels added
- uses ``daemon.json``

### CODE REVIEW

File: docker-centos/init.sh [hunk 1]
```diff
@@ -29,11 +29,7 @@ do
 done
 
 exec /usr/bin/docker-current daemon \
-          --authorization-plugin=rhel-push-plugin \
-          --add-runtime oci=/usr/libexec/docker/docker-runc-current \
-          --default-runtime=oci \
-          --containerd /run/containerd.sock \
-          --exec-opt native.cgroupdriver=systemd \
+          --config-file=/etc/docker/daemon.json \
```
Comment (giuseppe) [inline docker-centos/init.sh]: should the last \ be dropped?

File: docker-centos/Dockerfile [hunk 2]
```diff
@@ -1,12 +1,25 @@
...
+yum install --setopt=tsflags=nodocs -y --nogpgcheck docker-rhel-push-plugin && \
+rpm -V docker-rhel-push-plugin && \
+mkdir -p /exports/hostfs/etc/ && \
+yum clean all
 
 ADD init.sh /usr/bin
 
 # system container
 COPY service.template tmpfiles.template config.json.template /exports/
+COPY daemon.json /exports/hostfs/etc/
```
Comment (giuseppe) [inline docker-centos/Dockerfile]: this should be `COPY daemon.json /exports/hostfs/etc/docker/`

**Comment by giuseppe** [comment]
LGTM, I've tested this version and it works

**Comment by giuseppe** [comment]
@rh-atomic-bot r+  45fa94d

**Comment by ashcrow** [comment]
manually squashed

**Comment by giuseppe** [comment]
@rh-atomic-bot r+  	2e41902

**Comment by ashcrow** [comment]
@giuseppe ⬆️

**Pull request closed without merging**